app/src/main/java/com/hfad/sapper/ui/main/MainFragment.kt

```diff
@@ -2,29 +2,76 @@ package com.hfad.sapper.ui.main
 
 import androidx.lifecycle.ViewModelProvider
 import android.os.Bundle
+import android.util.Log
 import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import com.hfad.sapper.R
+import com.hfad.sapper.adapter.CellAdapter
+import com.hfad.sapper.adapter.ItemClickListener
+import com.hfad.sapper.data.Cell
+import kotlin.random.Random
 
-class MainFragment : Fragment() {
+const val FIELD_SIZE = 8
+const val CLEAR_CELL = -1
+const val MINE_CELL = 10
+const val TRIGGER_MINE_CELL = 11
```

Comment:
Cool, that you've moved these values to the constants. But you could do better: enum or sealed class.